src/data/judging/projectData.json

```diff
@@ -0,0 +1,240 @@
+[
+  {
```

Comment:
let's clear this for now since its dummy data

Reply:
I noticed you'd added `src/data/judging/projects.csv` to the `.gitignore`. Should these files be added too, or will these be used to post the actual project data to the site for judging?